### DISCUSSION TITLE
Can't download an engine

### DISCUSSION THREAD

**Discussion by Zacenator** [The-Wineskin-Project/WineskinServer#481] [Q&A] [answered]
Hi, when I download Wineskin Winery and click on the '+' to add an engine, nothing happens. The plus goes to grey, indicating it knows its been clicked, but no wine version come up. Is there any way to fix this?

**Comment by Gcenx** [comment, marked as answer]
You need to install via brew, sounds like you’ve failed to do this or your running an un supported version of macOS that had expired certs.
